docker/dockerfiles/allin1/allin1-ubuntu.Dockerfile

```diff
@@ -24,6 +24,7 @@ COPY ${LOCAL_REPO_PATH}/fs_brokers/apache_hdfs_broker/output/apache_hdfs_broker
 
 
 FROM artifacts-from-${ARTIFACT_SOURCE} as artifacts
+RUN rm -f /release/be_artifacts/be/lib/starrocks_be.debuginfo
```

Comment:
@kevincai  actually it's good to keep it in the allin1. 
in a case we have crash on BE deployed in k8s, we could download the coredump locally and run gdb with the debug info file in allin1 container

Reply:
it's alright, you can still pull the debuginfo from artifacts image. :)